dev/com.ibm.ws.jaxb_fat/fat/src/com/ibm/ws/jaxb/fat/LibertyJAXBOptionalFeatureTolerationTest.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxb.fat;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+
+/**
+ * Separate server startup needed because the goal is to test override of optional.jaxb-2.2 by jaxb-2.3
+ * Since both features are marked as singleton, fail of this test shows toleration failure
+ * server.xml has jaxb-2.3 and jpaContainer-2.1 features. jpaContainer-2.1 contains optional.jaxb-2.2 that we are testing it's override
+ */
+@RunWith(FATRunner.class)
+@SkipForRepeat("JAXB-2.3")
+public class LibertyJAXBOptionalFeatureTolerationTest extends FATServletClient {
```

Comment:
this test class could be combined with the other test class to remove duplication. Just use 2 `@Server` annotations in the above class, and start/stop the needed server in the test methods that use the servers instead of the before/afterClass

Reply:
These 2 tests are merged into LibertyJAXBTolerationTest creating 2 servers and moving their start/stop in test methods